ARCs/arc-0041.md

```diff
@@ -0,0 +1,107 @@
+---
+arc: 41
+title: ASA's vault of ASAs
+description: Application ID storing ASAs for another ASA
+author: Stéphane BARROSO (@SudoWeezy)
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/213
+status: Draft
+type: Standards Track
+category: Interface
+created: 2023-06-16
+requires: 3, 16
+---
+
+## Abstract
+This standard is inspired by <a href="https://eips.ethereum.org/EIPS/eip-6551">EIP-6551</a>
+The goal is to allow an ASA to detain & manage ASAs or Algo through a ”vault”.
+
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+
+### Glossary
+In the document we will use following syntax:
+- Main ASA -> ASA that detain & manage others ASAs
```

Comment:
The wording here is weird. "Main ASA" seems very vague and it took me awhile to really understand what it meant. I would suggest something like "Authority ASA". 

Also, the ASA itself can't hold or manage ASAs, but rather indicate a specific account can do those things. I think that should be made more clear.

Reply:
Good point, authority make more sense